spec/support/rspec_helpers.rb

```diff
@@ -0,0 +1,3 @@
+def select_option(css_selector, value)
+  find(:css, css_selector).find(:option, value).select_option
+end
```

Comment:
We'll have multiple helper files in this support folder so let's be more descriptive with the file name.

Also the rspec convention is to wrap this in a module, then import it in the `rspec_helper` file and do the following: 
```rb
config.include FormHelpers, type: :feature
```
```rb
module FormHelpers
  def select_option(css_selector, value)
    find(:css, css_selector).find(:option, value).select_option
  end
end
``` 
That way every feature spec will have access to it.